src/graphql/types/UsersConnection/index.ts

```
@@ -0,0 +1,18 @@
+import { builder } from "~/src/graphql/builder";
+import { usersTable } from "~/src/drizzle/tables/users";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)